README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# HW 01
+
+### Introduction
+
+![-](https://user-images.githubusercontent.com/67297411/85501690-7d090680-b5ab-11ea-9c45-1364676d9eda.png)
```

Comment:
The photo insertion is nicely done! I just want to suggest another way to insert photo( one which you might have already known:-) ). You can upload your photo to your repository and then use the code ![name]("the path of the photo in your repository")